### ISSUE TITLE
Rewrite description of other packages from @wolski review

### ISSUE THREAD

**Issue by ewallace** [ewallace/tidyqpcr#126]
Software review by @wolski https://github.com/ropensci/software-review/issues/470#issuecomment-985536546
points out that we have described other packages inaccurately.

This needs substantial effort and rewrites to `paper.md`.

> "Are there other R packages that accomplish the same thing? If so, how does yours differ or meet our criteria for best-in-category?"

> You explicitly mention the package qpcR and pcr.

> Your write "qPCR analysis package relying on an object-oriented approach using S4 classes."
> I am not a big fan of APIs with "@" either. But are you sure you are talking about qpcR? I checked the CRAN page and observed no dependency on the "methods". I downloaded the tar.gz and searched the sources for the "setClass" and "@" string, and there are no hits. "qpcR" is not written in S4.

We need to correct this, they use S3 classes not S4 classes.

> Furthermore, why do you say that "available packages either assume extensive prior R knowledge"?
> Does this indicate a GUI for tidyqpcr; a shiny application? Or can I accomplish the same task with tidyqpcr with fewer lines of code than when using the PCR package? Finally, the vignettes you include are not concise; on the contrary, you heavily use ggplot2 in the vignette, while ggplot2 does not have a flat learning curve.

We should refocus away from other packages: tidyqpcr is really easy if you know tidyverse. That means users can do customisable analysis & visualisation in few lines of code. To some extent "tidyverse is a qPCR package for tidyverse users". We also aim to be a gateway to the utility of the tidyverse for qPCR users. Can we explain that better?

For me (@ewallace) I found the other packages we mention impossible because they didn't expose the data in a data frame that I understand. I am not a typical user, so we need to find a better way of saying that, that is generous to these other carefully-written packages.

> Finally, you say "or lack extensive documentation".
> Both of the packages you mention are, in my opinion, well documented, and both are published in peer-reviewed journals:
> https://peerj.com/articles/4473/
> https://academic.oup.com/bioinformatics/article/24/13/1549/238435
> I am not claiming that this guarantees their utility but documented they are.

We need to revisit this. Or if we are referring to _tutorials_ that tidyqpcr now has, we need to say something.

> Furthermore, a web search for R and qPCR points to several more R packages that you do not mention:
> https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-296
> https://www.bioconductor.org/packages/release/bioc/html/HTqPCR.html

> I feel that you did not address the points in
> https://devguide.ropensci.org/policies.html#overlap sufficiently, and I would welcome a critical comparison.

> I will have a closer look at the three first points during the next week, but I have a few observations regarding:
> "Actively maintained while alternatives are poorly".

> qPCR has a release history of 14 years; it has one reverse depends and two reverse imports, and four reverse suggests. On the other hand, the pcr package has a release history of 4 years and an active github repository.

In summary, we have to do a more thorough review of these other packages including their documentation, then address the overlap more thoroughly in the review thread.

My preference is to say no negative things about other packages, and focus on the positives of tidyqpcr.

**Comment by DimmestP** [comment]
Draft response in review thread:

tidyqpcr was created to improve the quality of qPCR experiments using programming and experimental best-practices whilst being accessible to an audience that is unfamiliar with R data analysis. tidyqpcr follows the tidy data paradigm to ensure each step is human and machine readable. It also fully integrates into the wider collection of general data analysis packages provided by the tidyverse. This allows new users to get to grips with the basic pipeline quickly and expert users to develop customised analysis pipelines & visualisations.

Several other open-source libraries for qPCR analysis are available but not with the same aims as tidyqpcr. HTqPCR, ReadqPCR and NormqPCR have similar (and in some respects greater) functionality than tidyqpcr but follow an object oriented framework using S4 classes which does not align with our aims. qpcR (Spiess, 2018), which uses S3 classes not S4 as we previously stated, also uses an object oriented framework. PlateDesigner encourages better experimental design in qPCR experiments but does not support high-throughput experiments that use 384/1536 well plates. Finally, pcr (Ahmed & Kim, 2018) aligns most closely with tidyqpcr but its function inputs are not in a tidy format. Although many alternatives are available, non align with tidyqpcr’s aim to improve the quality of qPCR experiments from plate design to plotting results.

**Comment by ewallace** [comment]
My edits included below:

tidyqpcr was created to improve the quality of qPCR experimental design and analysis whilst being accessible to an audience that is new to R data analysis. Every data object in tidyqpcr is a tidy data frame, which makes the data both human and machine readable, enabling new users to get to grips with tidyqpcr features quickly. Following the tidy data paradigm integrates tidyqpcr into the wider collection of data analysis packages provided by the tidyverse. Tidyverse integration enables users to develop customised analysis pipelines and visualisations including integrating with other tidy data.

Several other open-source libraries for qPCR analysis are available with distinct aims. HTqPCR (Dvinge 2009), ReadqPCR/NormqPCR (Perkins et al., 2012), and qpcR (Spiess, 2018) have similar and in some respects greater functionality than tidyqpcr, but follow object oriented approaches which do not align with our aims. PlateDesigner encourages better experimental design in qPCR experiments but does not support 384- or 1536-well plates. Finally, pcr (Ahmed & Kim, 2018) aligns most closely with tidyqpcr but its function inputs are not tidy data frames. Although many alternatives are available, tidyqpcr's aims and approach are distinct: to improve the quality of qPCR experiments from plate design to analysis, by exposing all data in a consistent tidy format that integrates with the tidyverse.

**Comment by ewallace** [comment]
@DimmestP let's discuss the paper.md file too.
